tests/bearlib/languages/documentation/documentation_extraction_testdata/doxygen.py

```diff
@@ -2,35 +2,42 @@
 #  Documentation for this module.
 #
 #  More details.
+
 def func():
     pass
 
 ## Documentation for a class.
 #
 # More details.
+
```

Comment:
nope; wrong per https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html#pythonblocks

(and also this fixup *must* undo the bad changes in the previous patch to default.py : https://github.com/coala/coala/pull/4549#pullrequestreview-52493280 )